### DISCUSSION TITLE
Reverb issue after recording

### DISCUSSION THREAD

**Discussion by emmajanehopper** [ExistentialAudio/BlackHole#717] [Q&A]
Hello, and thanks so much for some great software.
I've been having an issue when recording audio. I'm recording news segments off of FB live/YouTube/radio streams using Black Hole through Pro Tools First. I have my PT playback set the 16 ch Black Hole, my input on sound in system preferences set to the black hole input I've set up (built in mic on top, Black Hole below it, Drift correction on Black Hole only) and my output on sound in system preferences set to my multi-output device with built in output on top, black hole ch 16 on bottom, drift correction on black hole (I've also tried with drift correction on built in output). With this set up, every time I play what I've recorded the recording has a strong reverb on it! I have no idea what's going on. My PT input on the track is In ch 1-2 and my output is Out ch 1-2. Please help! I can provide screenshots if needed.

**Comment by devinroth** [comment]
Screenshots are useful and if you have the audio for me to look at will help too.

**Comment by saburstow** [comment]
Hi, I have been having exactly the same reverb problem with recorded audio, except using Quicktime to record audio rather than ProTools. Any help gratefully received!
